src/content/structured/get-started/install-components.mdx

```diff
@@ -56,3 +56,7 @@ To use the components in a particular framework, follow the framework instructio
   </ic-link>
   .
 </p>
+
+## Using 3rd party libraries alongside ICDS
+
+If you can't find a suitable component in ICDS, then you can use one from a 3rd-party library in your project. Use the library's theming and customisation options to style the 3rd-party component using the [ICDS styles](/styles) so that they look and feel as similar as possible on the page. Make sure to check for new component releases from ICDS regularly so you can replace the 3rd-party components with ICDS ones when they are released.
```

Comment:
```suggestion
If you can't find a suitable component in ICDS, then you can use one from a third-party library in your project. Use that library's theming and customisation options to style the third-party component following the [ICDS styles](/styles) so that they look and feel as similar as possible on the page. Make sure to check for new component releases from ICDS regularly so you can replace the third-party components with ICDS ones when they are released.
```